script/QA/QA_22.txt

```diff
@@ -51,277 +57,313 @@ so please skip this section and go back to the main story.
 }
 [sha:096385fb702b5dd2b80fa7da0d394cd02b8e96f6]{
 -- Page 236, Offset 43752.
 -- 『光体』状態の真祖では空想具現化は
-The True Ancestor in the 'Luminary' state can
+A True Ancestor in a 'Luminary' state can%{n}
+only
 }
 [sha:93def1e370938010fbc01d0dbd86534acf180a53]{
 -- Page 236, Offset 43753.
 -- 大雑把なものしかできません。
-only produce a rough version of a Marble Phantasm.
+ produce a rough sketch of a Marble%{n}
+Phantasm.
 }
 [sha:0633c3074bea18267440c83dfffd931ff313152b]{
 -- Page 237, Offset 43754.
 -- 出力が大きくなった分、きめ細やかな空想を
-The larger the output, the harder it is to
+The larger the output,
 }
 [sha:53afbfbb25032d1bf309f243800fe59c8209dd8b]{
 -- Page 237, Offset 43755.
 -- 構築できなくなったのです。
-meticulously construct a phantasm accurately.
+ the harder it becomes to create%{n}
+finer details on the phantasms.
 }
 [sha:6c466f2664c7c1a8e412544fcf41d60d642d18dc]{
 -- Page 238, Offset 43756.
 -- 『光体』状態の彼女にできる事は、
-The things Arcueid can do in her 'Luminary' state
+The things Arcueid is capable of in her%{n}
+'Luminary' state
 }
 [sha:63b1f1fc140afd7c4ca8f3820a2b8eb5108c4453]{
 -- Page 238, Offset 43757.
 -- 超広範囲の魔眼と、
-include the use of her Ultra-wide Mystic Eyes,
+ include the use of her%{n}
+Ultra-wide Mystic Eyes―――
 }
 [sha:35c1cd278b8c1d98e24d409b2205d7379a983355]{
 -- Page 239, Offset 43758.
 -- 自身に向けられる攻撃をキャンセルする空想無量化と、
-canceling attacks directed at her with Imaginary Nullification,
+Canceling attacks directed at her using Phantasmal%{n}
+Nullification,
 }
 [sha:eda8f1c830acb99d5334fdc67df9d0e4598f7232]{
 -- Page 239, Offset 43759.
 -- 地熱操作、大気操作、といったものになります。
-manipulate geothermal heat, the atmosphere, and so on.
+ the manipulation of geothermal heat,
+ the%{n}manipulation of the atmosphere,
+ and so on.
 }
 [sha:54ef7834de68b5062217cc713a4a0ea63669fb80]{
 -- Page 240, Offset 43760.
 -- ようは自然の脅威です。これを、
-In short, it has become a threat to nature. She used to be
+In short, she has become a force of nature.
 }
 [sha:66cbe9bcd368bd108124cb3f2e7ead2a5147cdaa]{
 -- Page 241, Offset 43761.
 -- 『意識的に、指向性のものとして』
-a woman that decided her actions
+She used to be a woman who used this force%{n}
 }
 [sha:68eb344912cbe1a2a1ddd01476fa1c9d3720b8f1]{
 -- Page 241, Offset 43762.
 -- 使っていた普段の彼女と、
-'consciously and meticulously'.
+consciously, in accordance with her directive.
 }
 [sha:7a23a4a99cfe874c939d5df7a8697f83d19cb404]{
 -- Page 242, Offset 43763.
 -- 『無意識に、反応として』
-But the woman you see now can only act
+But now she wields it unconsciously, in%{n}
 }
 [sha:0650c76a1e45ff1d776a3e69eafbd69a2e51181a]{
 -- Page 242, Offset 43764.
 -- 使う事しかできない今の彼女。
-'unconsciously and reactively'.
+accordance with her impulses.
 }
 [sha:e845960c9e6ea1c15a8abd4e82163c05da413942]{
 -- Page 243, Offset 43765.
 -- どちらが生命として優れているかは、
-As for which one is better for humanity,
+As for which is better for life on this%{n}
+planet,
 }
 [sha:6f1943c0a568691554b6ecfc97b9db5083b17c6f]{
 -- Page 243, Offset 43766.
 -- あなたの嗜好でご判断いただければ。
-you can decide that based on your own preference.
+ it is up to you to decide.
```

Comment:
Well, this one needs to be restructured so that the problem in the latter sentence is amended and they link correctly.
>There is her usual self who employs nature consciously and with directive, and then there is how she is now, wielding it unconsciously and by impulse alone.

>Which one of these is the superior lifeform is something you'll have to judge by your own criteria.